<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecurityParamSetDialog</class>
 <widget class="QDialog" name="SecurityParamSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="CommonPushButton" name="pushButtonSecParaSetSave">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>380</y>
     <width>200</width>
     <height>96</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonSecParaSetReturn">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>480</y>
     <width>200</width>
     <height>96</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QFrame" name="framePanel">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>12</y>
     <width>700</width>
     <height>580</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frameSecuritySetting">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>12</y>
     <width>700</width>
     <height>580</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>22</y>
     <width>510</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>安防</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_securityset_1">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>85</y>
     <width>278</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24px;</string>
   </property>
   <property name="text">
    <string>进入延时（ 秒）</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_securityset_2">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>155</y>
     <width>278</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24px;</string>
   </property>
   <property name="text">
    <string>退出延时（秒）</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_securityset_3">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>225</y>
     <width>278</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24px;</string>
   </property>
   <property name="text">
    <string>重报间隔（秒）</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_securityset_4">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>295</y>
     <width>278</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24px;</string>
   </property>
   <property name="text">
    <string>防区8为门铃</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_securityset_5">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>365</y>
     <width>278</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24px;</string>
   </property>
   <property name="text">
    <string>发送图片到中心</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditEnterDelay">
   <property name="geometry">
    <rect>
     <x>322</x>
     <y>85</y>
     <width>422</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;
font: 24px</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditExitDelay">
   <property name="geometry">
    <rect>
     <x>322</x>
     <y>155</y>
     <width>422</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;
font: 24px</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditRepeatAlarmFilterInterval">
   <property name="geometry">
    <rect>
     <x>322</x>
     <y>225</y>
     <width>422</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;
font: 24px</string>
   </property>
   <property name="text">
    <string>10</string>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonSecParaSetDefense">
   <property name="geometry">
    <rect>
     <x>322</x>
     <y>295</y>
     <width>422</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24px</string>
   </property>
   <property name="text">
    <string>否</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonSecParaSetSendPic">
   <property name="geometry">
    <rect>
     <x>322</x>
     <y>365</y>
     <width>422</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24px</string>
   </property>
   <property name="text">
    <string>否</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommonPushButton</class>
   <extends>QPushButton</extends>
   <header>commonpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>mylineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButtonSecParaSetSave</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
